Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
True. I think somebody had worked on making it possible to view the merged Fusion code in the configuration module in the backend. But even if we had that, I doubt it would ease the pain so much for beginners.
If we keep the magic, we have to make a really good job explaining it.

So do we agree on keeping the generator, but adjusting its output to the new pattern?
If yes, then how to proceed with it? If we just replace the current default generator, it would be a hell of a breaking change for current projects.
Maybe we rename the old generator to smth like LegacyPrototypeGenerator, and make it the default until 4.0?